elastic/tox.ini

```diff
@@ -46,6 +46,7 @@ setenv =
     6.3: ELASTIC_IMAGE=6.3.2
     ; EOL 2020-02-23
     6.4: ELASTIC_IMAGE=6.4.2
+    7.2: ELASTIC_IMAGE=7.2.0
```

Comment:
This line sets the `ELASTIC_IMAGE` environment variable when running the tox env 7.2 but this env doesn't yet exist.
You also have to add an entry on line 8 so that tox will run this env.

```
py{27,37}-{5.4,5.5,5.6,6.0,6.1,6.2,6.3,6.4,7.2}
```